docs/conf.py

```diff
@@ -138,19 +138,13 @@
 # a list of builtin themes.
 #
 
-html_theme = "qiskit_sphinx_theme"
+html_theme = "qiskit-ecosystem"
+html_title = f"{project} {release}"
 
-html_logo = 'images/logo.png'
+#html_logo = 'images/logo.png'
```

Comment:
This should be deleted. It's better to not have commented out code: https://bytedev.medium.com/code-comment-anti-patterns-and-why-the-comment-you-just-wrote-is-probably-not-needed-919a92cf6758#:~:text=Avoid%20commenting%20out%20code%20and,to%20another%20team%20member%27s%20work.

Reply:
Okay! Thank you Eric!